src/chrome/content/scaffold/scaffold.xul

```diff
@@ -42,6 +42,14 @@
 		<key id="save" modifiers="accel" key="S" oncommand="Scaffold.save()"/>
 	</keyset>
 
+	<popupset>
+		<menupopup id="testing-context-menue">
+			<menuitem label="&scaffold.testing.edit.import;" tooltiptext="Edit the input data for the current test" oncommand="Scaffold.editImportFromTest()"/>
+			<menuitem label="&scaffold.testing.copyToClipboard;" tooltiptext="Copy the URL or data for the current test to the clipboard" oncommand="Scaffold.copyToClipboard()"/>
```

Comment:
When is this copying "data"? That's for non-web translators? What does it copy?

Reply:
Okay, this is now fixed. Please have a look at the new version.